Angadrandhawa1 alertresponse recovery update (#1311)
SumoLogic/sumologic-documentation: docs/alerts/monitors/alert-response.md

* **F** - opens the playbook associated with this Monitor. Playbooks allow admins to codify tribal knowledge for an on-call so they know what exactly to do when they receive an alert.<br/> ![playbook example.png](/img/monitors/playbook-example.png)<br/>
    You have the option to provide a playbook when creating a Monitor, as shown in the below image. Markdown is supported.<br/>  ![Montor playbook input.png](/img/monitors/monitor-playbook-input.png)
* **G** - opens the Monitor that generated this alert.
* **H** - resolves the Alert. This will also resolve the Monitor that generated the alert. The Monitor will fire again when the alert condition is met. <br/>
:::note

> Sumo Logic also resolves the alert automatically, when the recovery condition defined on the monitor is met. This behavior is not configurable (i.e., you cannot prevent Sumo Logic from disabling a monitor). Although technically you can set a recovery condition that will never allow Sumo Logic to recover a monitor, this is not recommended because it will suppress unrelated alerts from getting fired.

Reviewer: `you cannot prevent Sumo Logic from disabling a monitor` -> `you cannot prevent Sumo Logic from resolving a monitor`

* **F** - opens the playbook associated with this Monitor. Playbooks allow admins to codify tribal knowledge for an on-call so they know what exactly to do when they receive an alert.<br/> ![playbook example.png](/img/monitors/playbook-example.png)<br/>
    You have the option to provide a playbook when creating a Monitor, as shown in the below image. Markdown is supported.<br/>  ![Montor playbook input.png](/img/monitors/monitor-playbook-input.png)
* **G** - opens the Monitor that generated this alert.
* **H** - resolves the Alert. This will also resolve the Monitor that generated the alert. The Monitor will fire again when the alert condition is met. <br/> :::note

> Sumo Logic also resolves the alert automatically, when the recovery condition defined on the monitor is met. This behavior is not configurable (i.e user cannot prevent Sumo Logic from disabling a monitor). You can technically set a reovery condition that will never allow Sumo Logic to recover a monitor, but it is not recommended since it supress some alert from getting fired.

Reviewer 2: ```suggestion
Sumo Logic also resolves the alert automatically, when the recovery condition defined on the monitor is met. This behavior is not configurable (i.e user cannot prevent Sumo Logic from disabling a monitor). You can technically set a recovery condition that will never allow Sumo Logic to recover a monitor, but it is not recommended since it suppress some alert from getting fired.
```

> ::: <br/>![alert page sep 23.png](/img/monitors/alert-page.png)

Reviewer 2: @angadrandhawa1 if my suggestion looks ok to you, you can click the "Commit suggestion" button.

> * **H** - resolves the Alert. This will also resolve the Monitor that generated the alert. The Monitor will fire again when the alert condition is met. <br/> :::note
> Sumo Logic also resolves the alert automatically, when the recovery condition defined on the monitor is met. This behavior is not configurable (i.e user cannot prevent Sumo Logic from disabling a monitor). You can technically set a recovery condition that will never allow Sumo Logic to recover a monitor, but it is not recommended since it suppress some alert from getting fired.
> ::: <br/>![alert page sep 23.png](/img/monitors/alert-page.png)

Reviewer 2: ```suggestion
* **H** - resolves the Alert. This will also resolve the Monitor that generated the alert. The Monitor will fire again when the alert condition is met. <br/>
:::note
Sumo Logic also resolves the alert automatically, when the recovery condition defined on the monitor is met. This behavior is not configurable (i.e., you cannot prevent Sumo Logic from disabling a monitor). Although technically you can set a recovery condition that will never allow Sumo Logic to recover a monitor, this is not recommended because it will suppress unrelated alerts from getting fired.
::: <br/>![alert page sep 23.png](/img/monitors/alert-page.png)
```
